setup.py

```diff
@@ -14,8 +14,10 @@
       license='MIT',
       classifiers=[
           'License :: OSI Approved :: MIT License',
-          'Programming Language :: Python :: 2',
-          'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3.6',
+          'Programming Language :: Python :: 3.7',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
```

Comment:
When are these used? Remove all programming language classifiers?

Reply:
Please keep the programming language classifiers, they let users know what python versions the package supports.

The classifiers are use by the packaging system c.f. [Packaging Python Projects](https://packaging.python.org/tutorials/packaging-projects/#classifiers) 

When you look at a package distribution site entry for [bincopy](https://pypi.org/project/bincopy/) you will see the license and supported languages, i.e. in the lower left corner.

You may also want to refer to [Dropping support for older Python versions](https://packaging.python.org/guides/dropping-older-python-versions/) with respect to the automated fallback mechanism.

Reply:
Isn't ``python_requires='>=3.6'`` enough? The classifiers are just duplicated information.

Reply:
With respect to automated fallback the `python_requires='>=3.6'` is sufficient.